cmd/copyUtil_test.go

```diff
@@ -29,27 +29,39 @@ type copyUtilTestSuite struct{}
 
 var _ = chk.Suite(&copyUtilTestSuite{})
 
-func (s *copyUtilTestSuite) TestUrlIsContainerOrShare(c *chk.C) {
+func (s *copyUtilTestSuite) TestUrlIsContainerOrBlob(c *chk.C) {
 	util := copyHandlerUtil{}
 
 	testUrl := url.URL{Path: "/container/dir1"}
-	isContainerOrShare := util.urlIsContainerOrShare(&testUrl)
-	c.Assert(isContainerOrShare, chk.Equals, false)
+	isContainer := util.urlIsContainer(&testUrl)
+	c.Assert(isContainer, chk.Equals, false)
 
 	testUrl.Path = "/container/dir1/dir2"
-	isContainerOrShare = util.urlIsContainerOrShare(&testUrl)
-	c.Assert(isContainerOrShare, chk.Equals, false)
+	isContainer = util.urlIsContainer(&testUrl)
+	c.Assert(isContainer, chk.Equals, false)
 
 	testUrl.Path = "/container/"
-	isContainerOrShare = util.urlIsContainerOrShare(&testUrl)
-	c.Assert(isContainerOrShare, chk.Equals, true)
+	isContainer = util.urlIsContainer(&testUrl)
+	c.Assert(isContainer, chk.Equals, true)
 
 	testUrl.Path = "/container"
-	isContainerOrShare = util.urlIsContainerOrShare(&testUrl)
-	c.Assert(isContainerOrShare, chk.Equals, true)
+	isContainer = util.urlIsContainer(&testUrl)
+	c.Assert(isContainer, chk.Equals, true)
 
 	// root container
 	testUrl.Path = "/"
-	isContainerOrShare = util.urlIsContainerOrShare(&testUrl)
-	c.Assert(isContainerOrShare, chk.Equals, true)
+	isContainer = util.urlIsContainer(&testUrl)
+	c.Assert(isContainer, chk.Equals, true)
+}
+
+func (s *copyUtilTestSuite) TestIPIsContainerOrBlob(c *chk.C) {
+	util := copyHandlerUtil{}
+
+	testIP := url.URL{Host: "127.0.0.1:8256", Path: "/account/container"}
```

Comment:
Need to also test one of the form /account/container/virtualdir  Why, since without that test we have not tested the boundary condition of where (with how many slashes) the IP-form of the logic transitions from returning true to returning false.